Dockerfile

```
@@ -1 +1 @@
-FROM python:3.9
+FROM python:3.11.5
```

This should be 3.11, not 3.11.5 to preserve the same specificity
```suggestion
FROM python:3.11
```